src/app.module.ts

```diff
@@ -0,0 +1,16 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { DataSource } from 'typeorm';
+
+import { AppController } from './controllers/app.controller';
+import { AppService } from './services/app.service';
+import config from 'config';
+
+@Module({
+  imports: [TypeOrmModule.forRoot(config as TypeOrmModuleOptions)],
```

Comment:
The ConfigModule can simplify this process - https://docs.nestjs.com/techniques/configuration